packages/common/src/hooks/purchaseContent/validation.ts

```diff
@@ -57,7 +59,11 @@ export const createPurchaseContentSchema = (
     )
     .refine(
       (data) => {
-        if (data[GUEST_CHECKOUT] && !data[GUEST_EMAIL]) {
+        if (
```

Comment:
its the way this entire flow was designed, but it's not the best that we share schemas across all the pages imo